src/serverlist.cpp

```diff
@@ -479,27 +481,30 @@ void CServerListManager::CentralServerQueryServerList ( const CHostAddress& Inet
 /* Slave server functionality *************************************************/
 void CServerListManager::StoreRegistrationResult ( ESvrRegResult eResult )
 {
+    if ( bConsoleLogging )
+    {
+        qInfo() << "Registration response received:" << svrRegResultToString ( eResult );
```

Comment:
That is interesting. Does qInfo() does the same as your ConsoleWriterFactory?

Reply:
If yes, I will clean up the code today with this regard.

Reply:
It's a standard Qt global - I don't know how they implement it.

Reply:
The only thing to watch out for is it adds its own spacing, formatting and newlines.  So a string variable is output with quotes -- you need to `"stuff.. " + value + " ..more stuff"` rather than `"stuff.." << value << "..more stuff"` to avoid the quotes, I think.  Not tried yet.

Reply:
Ok, I'll do the changes.

Reply:
qInfo() does not output on the Windows CMD if you start Jamulus -s -n on the Windows CMD. So we still have to use your ConsoleWriterFactory. Can you please changes this here, too?

Reply:
Oh - hm.  OK.  I'll set up a macro I think, to make life simpler in future.

Reply:
Hm, wait for your change. Do we need an output on Windows cmd at all? If not, qInfo would do the job... Whats your opinion?

Reply:
For future info:
https://doc.qt.io/qt-5/qtglobal.html#qInfo
> Under Windows, the message is sent to the console, if it is a console application; otherwise, it is sent to the debugger.
Maybe it's the _if it is a console application_ - did you run with `-n` from the console?

Reply:
yes, I tried it with the -n option

Reply:
OK, I tried various things to bring it into line and had no luck.  I don't know C++ or Qt well enough to know why...  I'll have another look later.

Reply:
No problem. Let's keep it as is.